### PR TITLE
rosetta submit api

### PR DESCRIPTION
**Pull request by fdmirza**

### CODE REVIEW

**Comment by fariedt** [comment]
This is ready to merge, @zone117x.

File: src/tests-rosetta/api.ts [hunk 1]
```diff
@@ -1012,6 +1012,41 @@ describe('Rosetta API', () => {
...
 
+  test('construction/submit', async () => {
+    const request: RosettaConstructionHashRequest = {
+      network_identifier: {
+        blockchain: RosettaConstants.blockchain,
+        network: RosettaConstants.network,
+      },
+      //unsigned transaction bytes
+      signed_transaction:
+        '0x80800000000400164247d6f2b425ac5771423ae6c80c754f7172b0000000000000000000000000000000b400011ae06c14c967f999184ea8a7913125f09ab64004446fca89940f092509124b9e773aef483e925476c78ec58166dcecab3875b8fab8e9aa4213179d164463962803020000000000051a1ae3f911d8f1d46d7416bfbe4b593fd41eac19cb00000000000003e800000000000000000000000000000000000000000000000000000000000000000000',
```
Comment (zone117x) [inline src/tests-rosetta/api.ts:1023]: These tx payloads should be generated by stacks-tx lib so that the inputs are comparable to the outputs below

**Review by zone117x — CHANGES_REQUESTED** [review]

**Comment by zone117x** [comment]
@fariedt should we add the corresponding openAPI.yaml paths for this PR?

File: src/tests-rosetta/api.ts (hunk 1 above)
Comment (fariedt) [reply src/tests-rosetta/api.ts]: I've pushed a fix for this, along with the openapi path for the submit endpoint.

**Review by zone117x — APPROVED** [review]
🎉

**Pull request merged**